elasticsearch/client.go

```diff
@@ -142,16 +142,15 @@ func newV8Client(
 	fn backoffFunc,
 ) (Client, error) {
 	c, err := esv8.NewClient(esv8.Config{
-		APIKey:               apikey,
-		Username:             user,
-		Password:             pwd,
-		Addresses:            addresses,
-		Transport:            transport,
-		Header:               headers,
-		RetryOnStatus:        retryableStatuses,
-		EnableRetryOnTimeout: true,
```

Comment:
Is this the default now?

Reply:
No, it has been replaced with `RetryOnError` (https://github.com/elastic/go-elasticsearch/pull/385/files). Given that we already define [status codes for retry](https://github.com/elastic/apm-server/blob/main/elasticsearch/client.go#L34-L38), but haven't yet provided any retry methods, I believe this is what comes closest to the previous behavior.

Reply:
I went back through my memory and found https://github.com/elastic/elastic-transport-go/pull/3, where I reviewed the transport change. It does retry timeouts by default now: https://github.com/elastic/elastic-transport-go/pull/3/commits/89919cddeccea10fb860e17a1f085ac9f237f7cf#r767996155